### PR TITLE
cockpit: some fixes on the test scripts and a new test README

### PR DESCRIPTION
**Pull request by KKoukiou**

### CODE REVIEW

File: pyanaconda/ui/cockpit/anaconda-webui/test/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+A test will print out the commands to access it when it fails in this
+way. You can log into a running test-machine using ssh.
+
+You can also put calls to `sit()` into the tests themselves to stop them
+at strategic places.
+
+That way, you can run a test cleanly while still being able to make
+quick changes, such as adding debugging output to JavaScript.
+
```
Comment (M4rtinK) [inline pyanaconda/ui/cockpit/anaconda-webui/test/README.md]: Wow, this all sounds very convenient! :)

**Review by M4rtinK — APPROVED** [review]
Looks very nice to me! :)

Especially some oft the debugging options sounds so very nice! It really shows that the cockpit test suite is pretty mature & powerful. :)

**Comment by KKoukiou** [comment]
I will merge this, regardless of the missing deps on the tests container, because it's already an improvement and has some fixes.

File: pyanaconda/ui/cockpit/anaconda-webui/Makefile.am [hunk 2]
```diff
@@ -60,11 +60,14 @@ bots:
 
 # FIXME: we need to download this from somewhere if not existing
 $(ISO_FILE):
-	curl https://download.fedoraproject.org/pub/fedora/linux/development/rawhide/Server/x86_64/os/images/boot.iso --output $(ISO_FILE)
+	curl -LO https://download.fedoraproject.org/pub/fedora/linux/development/rawhide/Server/x86_64/os/images/boot.iso -o $(ISO_FILE)
```
Comment (jkonecny12) [inline pyanaconda/ui/cockpit/anaconda-webui/Makefile.am:63]: I think you don't want to use `O` when you already have `o` option there.

File: pyanaconda/ui/cockpit/anaconda-webui/test/run-test [hunk 3]
```diff
@@ -16,8 +16,8 @@ pushd ../../../../
 # FIXME boot.iso on rawhide does not currently contain the new cockpit dependencies
 # This will change once we include this changes upstream and start building boot.iso with the new dependencies
 # Then we can safely remove this workaround
-test ! -e cockpit-ws*.rpm && curl --output /dev/null https://kojipkgs.fedoraproject.org//packages/cockpit/259/1.fc35/x86_64/cockpit-ws-259-1.fc35.x86_64.rpm
-test ! -e cockpit-bridge*.rpm && curl --output /dev/null https://kojipkgs.fedoraproject.org//packages/cockpit/259/1.fc35/x86_64/cockpit-bridge-259-1.fc35.x86_64.rpm
+test ! -e cockpit-ws*.rpm && curl -LO -o /dev/null https://kojipkgs.fedoraproject.org//packages/cockpit/259/1.fc35/x86_64/cockpit-ws-259-1.fc35.x86_64.rpm
+test ! -e cockpit-bridge*.rpm && curl -LO -o /dev/null https://kojipkgs.fedoraproject.org//packages/cockpit/259/1.fc35/x86_64/cockpit-bridge-259-1.fc35.x86_64.rpm
```
Comment (jkonecny12) [inline pyanaconda/ui/cockpit/anaconda-webui/test/run-test:20]: The same here.

File: .cockpit-ci/run [hunk 4]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+# This is the expected entry point for Cockpit CI;
+
+set -eu
+
+ ./autogen.sh && ./configure
```
Comment (jkonecny12) [inline .cockpit-ci/run:7]: Is the leading space here on purpose or is it just a typo?

**Review by jkonecny12 — COMMENTED** [review]
Looks good to me otherwise.

**Pull request merged**